### PR TITLE
Make FungibleAssetValue.RawValue public

### PR DESCRIPTION
**Pull request by dahlia**
See also: <https://github.com/planetarium/lib9c/pull/19#discussion_r472835551>.

### CODE REVIEW

File: Libplanet/Assets/FungibleAssetValue.cs [hunk 1]
```diff
@@ -27,7 +29,14 @@ namespace Libplanet.Assets
...
 
-        internal readonly BigInteger RawValue;
+        /// <summary>
+        /// The internal representation of the fungible asset.
+        /// </summary>
+        /// <remarks>
+        /// Since this is an internal representation, this does not guarantee forward compatibility.
+        /// Therefore, do not depend on this value for permanent uses but only for volatile data.
+        /// </remarks>
+        public readonly BigInteger RawValue;
```
Comment (longfin) [inline Libplanet/Assets/FungibleAssetValue.cs:39]: How can we restore `FungibleAssetValue` by this `RawValue`? (`FungibleAssetValue(Currency, BigInteger)`  seems still internal)

Comment (dahlia) [reply Libplanet/Assets/FungibleAssetValue.cs:40]: Added: b577a6dd78e0a07140f96fac1c067cc6b87620cf.

**Review by longfin — APPROVED** [review]

File: Libplanet/Assets/FungibleAssetValue.cs [hunk 2]
```diff
@@ -440,6 +450,16 @@ public static FungibleAssetValue Parse(Currency currency, string value)
             return new FungibleAssetValue(currency, sign, major, minor);
         }
 
+        /// <summary>
+        /// Restores a value from a <paramref name="rawValue"/> of a <paramref name="currency"/>.
+        /// </summary>
+        /// <param name="currency">The currency to create a value.</param>
+        /// <param name="rawValue">The <see cref="RawValue"/> to restore.</param>
+        /// <returns>The value restored value the given <paramref name="rawValue"/>.</returns>
```
Comment (limebell) [inline Libplanet/Assets/FungibleAssetValue.cs:458]: Maybe "The value resored from"?

Comment (dahlia) [reply Libplanet/Assets/FungibleAssetValue.cs:458]: Though I intended “restore﻿·d,” is it unnatural?

Comment (limebell) [reply Libplanet/Assets/FungibleAssetValue.cs:458]: I thought "value" used twice... was it intended?

Comment (dahlia) [reply Libplanet/Assets/FungibleAssetValue.cs:458]: Not intnded… 🙄 I am going to fix it. Thanks!

Comment (dahlia) [reply Libplanet/Assets/FungibleAssetValue.cs]: @limebell Amended!

File: Libplanet/Assets/FungibleAssetValue.cs [hunk 3]
```diff
@@ -27,7 +29,15 @@ namespace Libplanet.Assets
...
 
-        internal readonly BigInteger RawValue;
+        /// <summary>
+        /// The internal representation of the fungible asset.
+        /// </summary>
+        /// <remarks>
+        /// Since this is an internal representation, this does not guarantee forward compatibility.
+        /// Therefore, do not depend on this value for permanent uses but only for volatile data.
+        /// </remarks>
+        /// <seealso cref="FromRawValue(Currency, BigInteger)"/>
```
[the remark this answers is not in the document]
Comment (dahlia) [reply Libplanet/Assets/FungibleAssetValue.cs]: @earlbread Amended!

**Comment by dahlia** [comment]
@planetarium/libplanet Fixed typos.  Please review this again!

**Pull request merged**